.rubocop.yml

```diff
@@ -28,7 +28,8 @@ Layout/LineLength:
   Enabled: false
 Layout/SpaceAroundMethodCallOperator:
   Enabled: true
-
+Lint/DeprecatedOpenSSLConstant:
```

Comment:
This can perhaps be removed when @skpaterson's comment is addressed?

Reply:
Rubocop still complains about that after removing the test feature of SDK from the plural resource.
```zsh
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Lint/DeprecatedOpenSSLConstant (0.84)
For more information: https://docs.rubocop.org/en/latest/versioning/
```